docs/configuration.asciidoc

```diff
@@ -1269,7 +1269,7 @@ Example: `0ms`.
 [[config-central-config]]
 ==== `central_config` (added[1.8.0])
 
-When enabled, the agent will make periodic requests to the APM Server to fetch updated configuration.
+When enabled, the agent will make periodic requests to the APM Server (every 5 minutes) to fetch updated configuration.
```

Comment:
```suggestion
When enabled, the agent will make periodic requests to the APM Server to fetch updated configuration.
The frequency of the periodic request is driven by the `Cache-Control` header returned from APM Server/Integration, falling back to 5 minutes if not defined.
```

It would be good to link to https://www.elastic.co/guide/en/observability/current/apm-configure-agent-config.html#_apm_agent_configuration_options